docs/rules-engine/how-to-use/chrome-extension.md

```diff
@@ -0,0 +1,111 @@
+# Rules Engine - Chrome Extension tab
+
+The Otter Framework provides a [Chrome Extension](https://chromewebstore.google.com/detail/otter-devtools/aejabgendbpckkdnjaphhlifbhepmbne) to help you debug your Rulesets. 
+
+## Activate the debug mode
+
+The extension is only compatible with applications running with the debug mode of the Otter rule engine.
+You can activate it via the ``forRoot`` method of the rules engine module:
+
+```typescript
+import {NgModule} from '@angular/core';
+import {RulesEngineRunnerModule, RulesEngineDevtoolsModule} from '@o3r/rules-engine';
+import {AppComponent} from './app.component';
+
+@NgModule({
+    imports: [
+      RulesEngineDevtoolsModule,
+      RulesEngineRunnerModule.forRoot({debug: true}) // Activate rule engine debug mode
```

Comment:
It's not enough you need to have the communication service between the app and the extension as well
`inject(RulesEngineDevtoolsMessageService)` in the main.ts
and `{provide: OTTER_RULES_ENGINE_DEVTOOLS_OPTIONS, useValue: {isActivatedOnBootstrap: true}}`